src/Tinker.jl

```diff
@@ -12,21 +25,11 @@ mutable struct ImageContext{T}
     pandrag::Signal{Bool}
     zoomclick::Signal{Bool}
     rectselect::Signal{Bool}
+    shape::Signal{<:Shape} # Holds selection outline
+    rectview::Signal{<:AbstractArray} # holds rectangular region corresponding to outline (type?)
```

Comment:
Glad it worked. Do keep in mind that this Signal will only update when you replace the *container*, not the objects inside the container.